src/lib/getPostText.ts

```diff
@@ -1,10 +1,12 @@
 import * as Mastodon from 'tsl-mastodon-api';
-const mastodon = new Mastodon.API({access_token: 'PRZhmwmS5fpkXo442UE8SGHv8TL7XOiqjhpOh49heb0', api_url: 'https://mastodon.social/api/v1/'}); // access the Mastodon API using the access token.
+import { sourceMastodonAccount, sourceMastodonId, bskyAccount, giveawayStrings } from "./config.js";
+
+const mastodon = new Mastodon.API({ access_token: "", api_url: "https://mastodon.social/api/v1/"})
```

Comment:
it looks like the content needed for the bot is public, and no API key needed (well, at least until it DoSes mastodon.social)